dipy/io/stateful_tractogram.py

```diff
@@ -10,8 +10,9 @@
                                             PerArrayDict)
 import numpy as np
 
+import dipy
```

Comment:
Do you really need this import? I don't see this being used further down.

Reply:
Maybe I am limited by my knowledge of such a thing in python, but I needed for this condition:
`if isinstance(reference, dipy.io.stateful_tractogram.StatefulTractogram):`
Since it is within the class I could use a direct reference from within it own declaration and I could not import the class from the file declaring the class, so `import dipy` was needed just for that.

Reply:
Unless I am missing something, I think that `isinstance(reference, type(self))` should work.

Reply:
I feel like I should have known that, thx !